README.md

```diff
@@ -54,3 +54,190 @@ For a simple, gentler, introduction to Lagom, have a look at the [Hello World gi
+```
+
+##### Build Project
+
+```bash
+sbt clean docker:publishLocal
+```
+
+##### View Images
+
+```bash
+docker images
+```
+
+##### Deploy Projects
+
+Finally, you're ready to deploy the services. Be sure to adjust the secret variables and cassanda service address as necessary.
+
+```bash
+# Secret values, be sure to change them
+
+secret_bidding=youmustchangeme
+secret_item=youmustchangeme
+secret_user=youmustchangeme
+secret_search=youmustchangeme
+secret_web=youmustchangeme
+
+# Default addresses for reactive-sandbox
+
+export service_cassandra=_cql._tcp.reactive-sandbox-cassandra.default.svc.cluster.local
+export service_kafka=_broker._tcp.reactive-sandbox-kafka.default.svc.cluster.local
+export service_elasticsearch=_http._tcp.reactive-sandbox-elasticsearch.default.svc.cluster.local
+
+# Configure the services to allow requests to the minikube IP (Play's Allowed Hosts Filter)
+
+allowed_host="$(minikube ip)"
+
+# Deploy bidding-impl
+
+rp generate-kubernetes-deployment biddingimpl:1.0.0-SNAPSHOT \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$secret_bidding -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --pod-controller-replicas 2 \
+  --external-service "cas_native=$service_cassandra" \
+  --external-service "kafka_native=$service_kafka" | kubectl apply -f -
+
+# Deploy item-impl
+
+rp generate-kubernetes-deployment itemimpl:1.0.0-SNAPSHOT \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$secret_item -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --pod-controller-replicas 2 \
+  --external-service "cas_native=$service_cassandra" \
+  --external-service "kafka_native=$service_kafka" | kubectl apply -f -
+
+# Deploy user-impl
+
+rp generate-kubernetes-deployment userimpl:1.0.0-SNAPSHOT \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$secret_user -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --pod-controller-replicas 2 \
+  --external-service "cas_native=$service_cassandra" \
+  --external-service "kafka_native=$service_kafka" | kubectl apply -f -
+
+# Deploy search-impl
+
+rp generate-kubernetes-deployment searchimpl:1.0.0-SNAPSHOT \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$secret_search -Dplay.filters.hosts.allowed.0=$allowed_host" \
+  --pod-controller-replicas 2 \
+  --external-service "cas_native=$service_cassandra" \
+  --external-service "kafka_native=$service_kafka" \
+  --external-service "elastic-search=$service_elasticsearch" | kubectl apply -f -
+
+# Deploy webgateway
+
+rp generate-kubernetes-deployment webgateway:1.0.0-SNAPSHOT \
+  --generate-pod-controllers --generate-services \
+  --env JAVA_OPTS="-Dplay.http.secret.key=$secret_web -Dplay.filters.hosts.allowed.0=$allowed_host" | kubectl apply -f -
+
+# Deploy ingress for everything
+
+rp generate-kubernetes-resources \
+  --ingress-path-suffix '*' \
```

Comment:
As mentioned in my long comment on the main thread, the `--ingress-path-suffix '*'` flag should be removed.